mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -61,7 +64,9 @@ class RegressionMetrics @Since("1.2.0") (
       case (prediction, _) => math.pow(prediction - yMean, 2)
     }.sum()
   }
-
+  private lazy val SSy = predictionAndObservations.map {
```

Comment:
This should already be computed in the statistical summarizer. This can be declared like: 

```scala
private lazy val SSy = math.pow(summary.normL2(0), 2)
```